### PR TITLE
updates to identity pages

### PR DESCRIPTION
**Pull request by cpengilly**
several updates to identity directory, att template, etc.

### CODE REVIEW

File: pages/identity/atst-v1.mdx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+# Ethereum Attestation Service
+This guide covers the basics of Ethereum Attestation Service, including EAS contract addresses, how to read and write attestations, and indexing.
+
+## EAS Contract Addresses
+
+The [Ethereum Attestation Service](https://docs.attest.sh/docs/welcome) is deployed on these addresses:
+
+| Network         | Attestation Contract | Schema Registry Contract |
+| --------------- | - | - |
+| OP Sepolia | [0x4200000000000000000000000000000000000021](https://goerli-optimism.etherscan.io/address/0x4200000000000000000000000000000000000021) | [0x4200000000000000000000000000000000000020](https://goerli-optimism.etherscan.io/address/0x4200000000000000000000000000000000000020)
```
Comment (zainbacchus) [inline pages/identity/atst-v1.mdx:16]: Should be sepolia:

https://sepolia-optimism.etherscan.io/address/0x4200000000000000000000000000000000000021
https://sepolia-optimism.etherscan.io/address/0x4200000000000000000000000000000000000020

File: pages/identity/atst-v1.mdx [hunk 2]
```diff
@@ -0,0 +1,35 @@
+---
+title: Ethereum Attestation Service
```
Comment (zainbacchus) [inline pages/identity/atst-v1.mdx:2]: Defer to you: Should this be Contracts?

File: pages/identity/atst-v1.mdx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+lang: en-US
+description: Learn the basics of Ethereum Attestation Service, EAS contract addresses, and how to read and write attestations.
+---
+
+# Ethereum Attestation Service
+This guide covers the basics of Ethereum Attestation Service, including EAS contract addresses, how to read and write attestations, and indexing.
```
Comment (zainbacchus) [inline pages/identity/atst-v1.mdx:8]: Add: "[Ethereum Attestation Service ("EAS")](https://attest.sh/), an open-source public good that is included as a predeploy in the OP Stack."

File: pages/identity/building-dapps-with-attestations.mdx [hunk 4]
```diff
@@ -1,44 +1,47 @@
-import { Callout } from 'nextra/components'
+---
+title: Build Decentralized Identity Apps with Attestations
```
Comment (zainbacchus) [inline pages/identity/building-dapps-with-attestations.mdx:2]: Defer to you: Should this be About attestations?

**Comment by zainbacchus** [comment]
Added some comments, otherwise LGTM! Thank you!

File: pages/identity/atst-v1.mdx (hunk 1 above)
Comment (cpengilly) [reply pages/identity/atst-v1.mdx:16]: fixing now!

File: pages/identity/atst-v1.mdx (hunk 2 above)
Comment (cpengilly) [reply pages/identity/atst-v1.mdx:2]: done :)

File: pages/identity/atst-v1.mdx (hunk 3 above)
Comment (cpengilly) [reply pages/identity/atst-v1.mdx:8]: done :)

File: pages/identity/building-dapps-with-attestations.mdx (hunk 4 above)
Comment (cpengilly) [reply pages/identity/building-dapps-with-attestations.mdx:2]: Build Decentralized Identity Apps with Attestations

**Review by zainbacchus — APPROVED** [review]

**Pull request merged**